src/rust/ide/src/ide/integration.rs

```diff
@@ -988,6 +1005,11 @@ impl Model {
 
         debug!(self.logger, "Attaching visualization on {node_id}.");
         let visualization  = self.prepare_visualization(node_id)?;
```

Comment:
How big this data can be? What does it contain?

Reply:
Probably more than we need. I'm only adding the field `visualization_module`, which tells us which visualization was used. We don't care about e.g., the preprocessor at the moment.